CAIPs/caip-3.md

```diff
@@ -0,0 +1,68 @@
+---
+caip: 3
+title: Blockchain Reference for the EIP155 Namespace
+author: Simon Warta (@webmaster128), ligi <ligi@ligi.de>, Pedro Gomes (@pedrouid)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/issues/3, https://github.com/ChainAgnostic/CAIPs/pull/1
+status: Draft
+type: Standard
+created: 2019-12-05
+updated: 2020-01-16
+requires: 2
+---
+
+## Simple Summary
+
+This document is about the details of the EIP155 namespace and reference for CAIP-2.
+
+## Abstract
+
+In CAIP-2 a general blockchain identification scheme is defined. This is the
+implementation of CAIP-2 for EIP155 (Ethereum).
+
+## Motivation
+
+See CAIP-2.
+
+## Specification
+
+### EIP155 Namespace
+
+The namespace is called "eip155" as in [EIP155](https://eips.ethereum.org/EIPS/eip-155).
+
+#### Reference Definition
+
+The definition is delegated to EIP155. The format is an unsigned integer in decimal represenation and corresponds to `CHAIN_ID` of EIP155.
```

Comment:
typo
```suggestion
The definition is delegated to EIP155. The format is an unsigned integer in decimal representation and corresponds to `CHAIN_ID` of EIP155.
```

Reply:
Hmm, why can't I accept this change?

Reply:
hm - I have the button - but I cannot press it:

![Selection_326](https://user-images.githubusercontent.com/111600/72687895-af956b80-3b02-11ea-89d4-3c071ac26fd7.png)

Reply:
Done in 5740ce43f028367d34b885209e5a4d7a5917f669

Reply:
Done in 5740ce43f028367d34b885209e5a4d7a5917f669